src/components/ui/Collapsible/fragments/CollapsibleTrigger.tsx

```
@@ -0,0 +1,65 @@
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        d="M9 9V4.5M9 9H4.5M9 9 3.75 3.75M9 15v4.5M9 15H4.5M9 15l-5.25 5.25M15 9h4.5M15 9V4.5M15 9l5.25-5.25M15 15h4.5M15 15v4.5m0-4.5 5.25 5.25"
+      />
+    </svg>
+  );
+
+
+
+const CollapsibleTrigger = ({children, asChild=false}: CollapsibleTriggerProps) => {
+    const {rootClass,open,onOpenChange} = useContext(CollapsibleContext)
+      const toggleCollapse = () => onOpenChange && onOpenChange((p) => (!p))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)